components/Icons/Icon.tsx

```diff
@@ -1,89 +1,41 @@
-import SmileSVG from '@/public/assets/icons/icon_smile.svg';
 import CheckCircleSVG from '@/public/assets/icons/icon_check_circle.svg';
-import ScenarioSVG from '@/public/assets/icons/icon_scenario.svg';
-import ArrowSVG from '@/public/assets/icons/icon_arrow.svg';
 import RecorderSVG from '@/public/assets/icons/icon_recorder.svg';
 import BulbSVG from '@/public/assets/icons/icon_bulb.svg';
-import TwinkleSVG from '@/public/assets/icons/icon_twinkle.svg';
 import MenuSVG from '@/public/assets/icons/icon_menu.svg';
 import DangerSVG from '@/public/assets/icons/icon_alert_danger.svg';
 import SuccessSVG from '@/public/assets/icons/icon_alert_success.svg';
 import WarningSVG from '@/public/assets/icons/icon_alert_warning.svg';
 import InfoSVG from '@/public/assets/icons/icon_alert_info.svg';
 import CheckSVG from '@/public/assets/icons/icon_check.svg';
-import DiscordSVG from '@/public/assets/icons/icon_discord.svg';
 import ViewFullSVG from '@/public/assets/icons/icon_view_full.svg';
 import ViewShowSVG from '@/public/assets/icons/icon_view_show.svg';
 import ViewGridSVG from '@/public/assets/icons/icon_view_grid.svg';
 import ViewSplitSVG from '@/public/assets/icons/icon_view_split.svg';
-import ExpandSVG from '@/public/assets/icons/icon_expand.svg';
-import PlusSVG from '@/public/assets/icons/icon_plus.svg';
-import MinimizeSVG from '@/public/assets/icons/icon_minimize.svg';
 import PinSVG from '@/public/assets/icons/icon_pin.svg';
-import PackageSVG from '@/public/assets/icons/icon_package.svg';
-import CloudSVG from '@/public/assets/icons/icon_cloud_orange.svg';
-import FolderSVG from '@/public/assets/icons/icon_folder.svg';
-import FolderOpenSVG from '@/public/assets/icons/icon_folder_open.svg';
-import FileSVG from '@/public/assets/icons/icon_file.svg';
 
 const svgMap = {
-  star: <StarSVG />,
-  copy: <CopySVG />,
-  book: <BookSVG />,
-  backHome: <BackHomeSVG />,
-  slack: <SlackSVG />,
-  github: <GitHubSVG />,
-  gnbMenu: <GnbMenuSVG />,
-  close: <CloseSVG />,
-  closeSmall: <CloseSmallSVG />,
   openSelector: <OpenSelectorSVG />,
-  diamond: <DiamondSVG />,
   tool: <ToolSVG />,
-  messageSquare: <MessageSquareSVG />,
   messageSmile: <MessageSmileSVG />,
-  smile: <SmileSVG />,
   checkCircle: <CheckCircleSVG />,
-  scenario: <ScenarioSVG />,
-  arrow: <ArrowSVG />,
   recorder: <RecorderSVG />,
   bulb: <BulbSVG />,
-  twinkle: <TwinkleSVG />,
   menu: <MenuSVG />,
   danger: <DangerSVG />,
   success: <SuccessSVG />,
   warning: <WarningSVG />,
   info: <InfoSVG />,
   check: <CheckSVG />,
-  discord: <DiscordSVG />,
   viewFull: <ViewFullSVG />,
   viewShow: <ViewShowSVG />,
   viewGrid: <ViewGridSVG />,
   viewSplit: <ViewSplitSVG />,
-  expand: <ExpandSVG />,
-  plus: <PlusSVG />,
-  minimize: <MinimizeSVG />,
   pin: <PinSVG />,
-  package: <PackageSVG />,
-  cloud: <CloudSVG />,
-  folderClose: <FolderSVG />,
-  folderOpen: <FolderOpenSVG />,
-  file: <FileSVG />,
 };
 type SVGType = keyof typeof svgMap;
```

Comment:
Some icons were moved from `.svg` to `.tsx`. Is there a reason for this?

Reply:
Yes, that's correct. Icons are used in the Icon Component and Button Icon. To adjust the size of the Icon based on size such as `xs`, `lg`, etc., the icon's width and height should be updated accordingly. Currently, the SVG has fixed width and height.